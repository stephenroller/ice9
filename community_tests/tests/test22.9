# != with ints and bools									succeed

1 != 2;
true != true;
true != (true != false);
