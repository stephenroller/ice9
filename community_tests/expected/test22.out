line 5: syntax error near end of file
